server/routes/api/user.py

```diff
@@ -35,31 +37,45 @@ class ImportStatus(Enum):
     IMPORTED = 1
 
 
-def get_user_db():
+@dataclass
+class Import:
+    """Data type for import entry"""
+    status: int
```

Comment:
nit:  comment this uses ImportStatus enum

Reply:
Thanks! I will make this change in follow up PR